src/features/wallet/Wallet.tsx

```diff
@@ -242,7 +241,7 @@ const Wallet: FC<Props> = ({ signingKeys, refetchVoteSummaryData }) => {
               <Button
                 className="connect-btn"
                 onClick={() => {
-                  if (!initOnboard) setInitOnboard(true);
+                  connect().catch(console.error)
```

Comment:
with the onboard refactor, we dont need to re-init onboard, we can just call connect